docs/updates/release-notes.md

```diff
@@ -9,6 +9,25 @@ import ContentRef from '@site/src/components/ContentRef';
 
 # Latest Release Notes
 
+## 2.9.11 - 2022-06-20 - Release Notes Component, Enterprise Store Improvements
+
+### 🆕 New Feature
+
+- [Release Notes Component](https://docs.appcircle.io/integrations/managing-release-notes). You can create release notes with Publish Release Notes component.
+- [Enterprise Store](https://docs.appcircle.io/enterprise-appstore/customize-ent-store) Certificate Details added to Enterprise Store.
+- [Enterprise Store](https://docs.appcircle.io/enterprise-appstore/enterprise-reports) Detailed reports are added to Enterprise Store.
+- [Open API](https://api.appcircle.io/openapi/index.html?urls.primaryName=signing-identity) New API endpoints added to Certificate and Provisioning profiles upload.
+- [Open API](https://api.appcircle.io/openapi/index.html?urls.primaryName=build) New API endpoint added to start a build with provided environment variables.
+
+### :muscle: Improvement
+- [iOS Stack](https://docs.appcircle.io/infrastructure/ios-build-infrastructure) Monterey is upgraded to 12.4 for macOS agents.
+- Log window is improved. It is more performant and stable.
+- Added [FAQ section](https://docs.appcircle.io/troubleshooting-faq/common-issues#issues-in-connecting-to-the-repositories-with-ssh) for multiple SSH keys.
```

Comment:
@tosbaha I followed URL written in here. But it ends with "page not found" on second level click.

![2022-06-21_14-05](https://user-images.githubusercontent.com/7355362/174785196-ae007659-efc4-48c9-95ba-2e10e50b5271.png)